docs/rule/no-aria-hidden-body.md

```diff
@@ -0,0 +1,27 @@
+# no-aria-hidden-body
+
+:wrench: The `--fix` option on the command line can automatically fix some of the problems reported by this rule.
+
+The aria-hidden attribute should never be present on the `<body>` element, as it hides the entire document from assistive technology.
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+<body aria-hidden>
+```
+
+```hbs
+<body aria-hidden="true">
+```
+
+This rule **allows** the following:
+
+```hbs
+<body>
+```
+
+## References
```

Comment:
Can you add a reference link that explains `aria-hidden`?

Reply:
This isn't a WCAG reference, but this blog post introduced me to the value of aria-hidden.

https://heydonworks.com/article/aria-label-is-a-xenophobe

Reply:
done